contentful_cli/management.py

```diff
@@ -393,7 +419,12 @@ def construct_base_url(self):
         raise Exception('No suitable contentful-api-gateway endpoint for Contentful subdomain {}'.format(self.subdomain))
 
     def invoke(self, arguments, session, oauth_token, gateway_api_key, run=True):
-        expanded_path = self.template.format(**arguments)
+
+        templating_arguments = dict(arguments)
+        if self.environment_aware and arguments.get('environment_id'):
+            templating_arguments['space_id'] += '/environments/' + arguments['environment_id']
+
+        expanded_path = self.template.format(**templating_arguments)
```

Comment:
Mixing `.format` and `+= calls` in the same block of code seems a bit confusing. We should stick to one method of string concatenation and use that consistently.

It seems `+=` is only used in [one other place](https://github.com/Skyscanner/skyscanner-contentful-management/blob/e17a8eaeba8a31995925d396d5e3bfadc3d08d6b/contentful_cli/management.py#L425), whereas `.format` is regularly used throughout the file (even for single concatenations).

Reply:
To be honest I don't see why `+=` and `format` preclude each other, but I'll change it if you prefer. I could see an argument that it's a bit messy to draw the space_id from the new dictionary and the environment_id from the old dictionary.

I'll revise this to try to make the intent a bit clearer.